README.md

```diff
@@ -168,8 +169,7 @@ Now install the python packages.
 These are mostly installed via the `conda` command; `pip` is only used for packages not yet available via conda.
 
 ```
-conda install biopython nestly pyqt
-pip install ete3 scons
+conda install biopython nestly pyqt ete3 scons colorbrewer
```

Comment:
There was some reason we were using pip for this, but if this worked for you, I'm happy to simplify things.

Reply:
I decided you had it as pip because they didn't used to be in conda, but I'm not sure how sure I was of that.

Reply:
Let's run with it for now :+1: